src/index.js

```diff
@@ -1,18 +1,32 @@
 const login = prompt('Enter login');
 const password = prompt('Enter password');
 
-if (login === 'admin') {
-  if (passwrd === 'qwerty1234') {
-    alrt('Hello, admin');
+if (login === 'admin' && password === 'qwerty1234') {
+  alert ('Hello, admim');
+} else if (login === 'guest' && password === 'guest666') {
+    alert('Hello, guest');
+  } else if (login !== 'admin' && login !== 'guest') {
+   alert('Go away.');
+  } else {
+    alert('Incorrect info.');
```

Comment:
Please format code. In every nested code you have 2 spaces

`
const a = 2;
if (true) {
  console.log('2 spaces'();

  if (false) {
    console.log('4 spaces');
  }
}
`